mocks/tests/conftest.py

```diff
@@ -46,3 +48,8 @@ def custom_cosmology_data():
     custom_data['hubble_constant'] = 70.
     custom_data['T_CMB0'] = 2.5
     return custom_data
+
+
+@pytest.fixture
```

Comment:
When the fixture scope was `package` and I run the test locally there were no errors and it was totally alright but Travis raised an error for that!

Reply:
OK, package scope was new in pytest 3.7, Travis is probably using an older version by default. For now just use module scope.